src/OpenLoco/src/Ui/ScrollView.h

```diff
@@ -5,8 +5,10 @@
 namespace OpenLoco::Ui::ScrollView
 {
     constexpr uint8_t thumbSize = 10;
-    constexpr uint8_t barWidth = 11;
+    constexpr uint8_t barThickness = 11;
```

Comment:
Don't we typically annotate constants with k?

Reply:
yep and I've just noticed there are some constants in ScrollViewWidget.cpp that should be in ScrollView.h